### PR TITLE
Add some more tests

### PR DESCRIPTION
**Pull request by underscoredotspace**
I've started with colorContraster since it's fairly simple, but I'd love to see some Vue component tests from someone who knows what they're doing so I can get in on that in the future. 

Also, I watched today's stream and thought "oh, Suz must be using tabs" (after seeing the PR) and I switched my editor to tabs, but oops. I've submitted a second commit to fix this.

### CODE REVIEW

**Comment by underscoredotspace** [comment]
Lots of relevant discussion in the Tests issue (#5).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -122,9 +122,9 @@ It's really fine I promise.
 
 ### Tests
 
-There are currently no tests 😱 Yes this is a priority for me going forward! I like Tape for backend tests and Jest for front-end tests but I am open to suggestions!
+[Jest](https://facebook.github.io/jest/) 🃏 is used for testing, and the tests are saved beside the code being tested, in a folder called `specs`. **Vue Test Utils** is used to help make testing the Vue components easier and it has lots of [documentation](https://vue-test-utils.vuejs.org/) to help you out. 
 
-If you're struggling to set up tests without build tools like webpack / babel / gulp / vue-cli I am happy to talk to you about this.
+There is not much coverage yet, so please add some more tests. Writing tests is very good for your welbeing 😉, and maybe you'll find a little bug to fix! 🐛
```
Comment (underscoredotspace) [inline CONTRIBUTING.md]: I've tried to maintain tone-of-voice, but would welcome feedback here especially.

Comment (mtheoryx) [reply CONTRIBUTING.md]: I like it, a little cheeky, but it fits our tone :)

**Comment by mtheoryx** [comment]
Nice addition, and I'm glad we finally tracked down the Vue undefined issue (props to @escherina and @mpaarating on that one).

**Comment by noopkat** [comment]
This is amazing work @underscoredotspace I'm really stoked to have good quality and neat testing in this repo ✨ 

Also I am really warm and fuzzy about all the teamwork that's been going on in this repo as of late. Thanks for being awesome.

bitmoji

**Pull request merged**